db/seeds.rb

```diff
@@ -0,0 +1,253 @@
+#coupons
+BsCheckout::Coupon.create!(code: 'qwerty123', value: '51,1')
+
+#deliveries
+BsCheckout::Delivery.create!(name: 'Delivery Next Day!', interval: '3 to 7 days', price: '28,50')
+BsCheckout::Delivery.create!(name: 'Free!', interval: '10 days', price: '0')
+
+#countries
+BsCheckout::Country.create!(name: 'Afghanistan')
```

Comment:
You added those countries manually?oO

Reply:
Yeah )
I found a list of countries and then added it manually.

Reply:
There are few gems that includes all countries you can seed them from it https://github.com/hexorx/countries
Much faster then do it manualy